wagtail_bucketav/views.py

```diff
@@ -28,6 +28,9 @@ def handle_message(self, message, payload):
             logger.error("Received ping for unknown file key=%s", file_key)
             return
 
+        if isinstance(instance, BucketAVMixin):
+            instance.update_scan_status(file_scan_status)
+
```

Comment:
Issue: The intention was to handle this using a hook rather than needing to bake the functionality into the view